mars/dataframe/utils.py

```diff
@@ -431,3 +431,13 @@ def filter_index_value(index_value, min_max, store_data=False):
         f = f & (pd_index < max_val)
 
     return parse_index(pd_index[f], store_data=store_data)
+
+
+def wrap_exception(func):
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except NotImplementedError as e:
+            print(e)
```

Comment:
Seems uncovered here, should return NotImplemented instead, try to write ut to cover this line.